lib/express/websocket.js

```diff
@@ -0,0 +1,66 @@
+import _ from 'lodash';
+import url from 'url';
+
+
+const WEBSOCKET_SERVERS = {};
+let IS_UPGRADE_LISTENER_ASSIGNED = false;
+const PATHNAME_PREFIX = '/wd/ws';
```

Comment:
given that the `/wd/hub` prefix is deprecated in w3c, maybe we should just bite the bullet and call this `/ws`?